includes/admin/class-admin-settings.php

```diff
@@ -860,7 +860,7 @@ public static function get_field_description( $value ) {
 			$value           = ( isset( $value[ $description_key ] ) && ! empty( $value[ $description_key ] ) ) ? $value[ $description_key ] : '';
 
 			if ( ! empty( $value ) ) {
-				$description = '<p class="give-field-description">' . wp_kses_post( $value ) . '</p>';
+				$description = '<div class="give-field-description">' . wp_kses_post( $value ) . '</div>';
```

Comment:
@DevinWalker Setting field description will be wrap in `div` instead of `p` because `ul` was coming out from `p` and we also can not add `div` inside `p` if needed.